readthedocs/doc_builder/environments.py

```diff
@@ -315,8 +315,8 @@ def run(self):
                 cmd_stdout, cmd_stderr = out
```

Comment:
Alternatively, you could set defaults at the top -- plus a walrus!

```suggestion
                cmd_stdout, cmd_stderr = (out := ("", ""))
```

If it helps to default `cmd_stdout` to `""` instead of `None`.

Reply:
This overrides the real output

```
>>> out = (None, "foo")
>>> cmd_stdout, cmd_stderr = (out := ("", ""))
>>> cmd_stdout
''
>>> cmd_stderr
''
```